qiskit/visualization/gate_map.py

```diff
@@ -349,6 +335,82 @@ def plot_gate_map(
     num_qubits = config.n_qubits
     cmap = config.coupling_map
 
+    return plot_gate_map_data(
+        num_qubits,
+        mpl_data,
+        cmap,
+        figsize,
+        plot_directed,
+        label_qubits,
+        qubit_size,
+        line_width,
+        font_size,
+        qubit_color,
+        qubit_labels,
+        line_color,
+        font_color,
+        ax,
+    )
+
+
+def plot_gate_map_data(
+    num_qubits,
+    mpl_data,
+    cmap,
```

Comment:
In many matplotlib example code we use `cmap` for the color map object and thus this is bit confusing. Can you explicitly say coupling_map? `mpl_data` is also not clear to me. Perhaps `qubit_coordinates`?

Reply:
I kept the names as in the original implementation - sure I'll change according to your suggestion.